conductor/lib/uploader.py

```diff
@@ -46,7 +49,8 @@ def do_work(self, job, thread_int):
                 logger.error(message)
                 raise Exception(message)
         self.metric_store.set_dict('file_md5s', filename, current_md5)
-        return (filename, current_md5)
+        size_bytes = os.path.getsize(filename)
```

Comment:
Since the HTTPBatchWorker is already taking in the size as a request body, we could change how singlepart and multipart looks and include the size as part of the response for each file needed for upload (its a relatively easy change). FileStatWorker would be renamed to something else and use each file response payload (size) to setup the metric increments.

Reply:
Up to you.  You can leave as is.